tests/_conftest.py

```diff
@@ -47,6 +61,7 @@ def app(database):
     app = Flask(__name__)
 
     app.config['SQLALCHEMY_DATABASE_URI'] = DB_CONN
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
```

Comment:
What's your reasoning behind adding this config?

Reply:
Oh yes, essentially SQLAlchemy pollutes the log with deprecation warning messages because default behaviour is changing. I noticed it when I was running `pytest` with the stdout capture disabled. Naturally if you run it normally you don't see it.

As to why I set it to True, just because it preserved existing behaviour. That being said, it is not required by the library, and disabling it/leaving it alone works fine. So I'll remove this statement as well.

Reply:
Great, thanks for the context! I'm glad you flagged this, we should address deprecations in a separate issue.